internal/api/resolver_model_studio.go

```diff
@@ -2,30 +2,14 @@ package api
 
 import (
 	"context"
-	"time"
 
-	"github.com/stashapp/stash/internal/api/loaders"
 	"github.com/stashapp/stash/internal/api/urlbuilders"
 	"github.com/stashapp/stash/pkg/gallery"
 	"github.com/stashapp/stash/pkg/image"
 	"github.com/stashapp/stash/pkg/models"
 	"github.com/stashapp/stash/pkg/performer"
 )
 
-func (r *studioResolver) Name(ctx context.Context, obj *models.Studio) (string, error) {
-	if obj.Name.Valid {
-		return obj.Name.String, nil
-	}
-	panic("null name") // TODO make name required
```

Comment:
Name is required in the UI, but I don't think it's enforced anywhere. I will make a note to address that later.